client/src/components/Card/Card.css

```diff
@@ -0,0 +1,8 @@
+.slick-prev:before,
+.slick-next:before {
+  color: black;
+}
+
+.left-align-slick > .slick-list > .slick-track {    
```

Comment:
Just include this to left align the items in slider

`.slick-track { margin-left:0; }`

Reply:
Done.